src/nativeresources/processrc.awk

```diff
@@ -54,12 +56,14 @@ BEGIN {
         gsub(/" +"/, "", $0);
         # remove all terminating newlines from the string - the msgfmt fails on those
         # since it expects them to be at the end of the msgid as well
-        while(gsub(/\\n"/, "\"", $0)) {}
+        if (removeTerm == 1) {
```

Comment:
The stripping was necessary to make the internationalization work with the unix gettext API. Please note that this removes terminating newlines - the ones before the closing quote - not the ones in the middle of the strings.
However, this is likely not a problem anymore. Originally, we were generating numeric string ids. Now the string id is the english version of the resource string and so I think the newlines should be ok.
Could you please remove the parameter and the while below that removes the newlines?